admin/partials/page-json.php

```diff
@@ -86,6 +86,18 @@
 				</label>
 			</div>
 			<?php if ( ! empty( $selected_theme ) ) : ?>
+				<div class="theme-value-wrapper">
+				<?php if ( ! empty( $layout_columns ) ) : ?>
+					<div class="theme-value-row top">
+						<?php esc_html_e( 'Theme Columns', 'apple-news' ); ?>: <span><?php echo esc_html( $layout_columns ); ?></span>
```

Comment:
Since this is a number, we can be even more specific in our escaping here:
```suggestion
						<?php esc_html_e( 'Theme Columns', 'apple-news' ); ?>: <span><?php echo (int) $layout_columns; ?></span>
```